nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/ShutdownHook.java

```diff
@@ -66,7 +68,7 @@ public void run() {
         }
 
         runner.notifyStop();
-        System.out.println("Waiting for Apache NiFi to finish shutting down...");
+        System.out.println("Waiting for Apache NiFi to finish shutting down... pid=" + pid);
```

Comment:
Also recommend adjusting this message:
```suggestion
        System.out.printf("NiFi process [%d] shutdown in progress...%n", pid); 
```